### ISSUE TITLE
about the results

### ISSUE THREAD

**Issue by NerminSalem** [bobqywei/inpainting-partial-conv#8]
hope you are fine. i used the test file after adjusting it to work with one image and used the model u provided but didn't get good results as the video? do u think the model needs more training ?
![test](https://user-images.githubusercontent.com/26221951/51802216-5b89db00-2250-11e9-8f21-325187c01d0c.jpg)
thanks alot for your consistent help:)

**Comment by bobqywei** [comment]
@NerminSalem Hey good to see it's working. How many epochs did you train the model for? I think I trained for over 2 epochs on Places2 as well as additional finetuning (which you can set through the argparser). Even then, my final model did not perform well in all scenarios (complex images with lots of contrasting edges). If you have time to play around, you can try tweaking the loss functions as well (look over the original paper).

**Comment by NerminSalem** [comment]
I have used the model you put a link to in issue number 6

**Comment by bobqywei** [comment]
Oh I see. Again, my final model doesn't perform well in all scenarios. I believe it would require training for a longer duration and strategic fine-tuning, as the original paper mentioned much longer GPU hours (512 images). It seems very poor in this example though still. Does it perform similarly on other images?

**Comment by NerminSalem** [comment]
Yes i tried more than an image and i always got results that have aircrafts in the mask region
Is the model u provided is the final model and i think of training to 4 epoches. Do u think the results will be of any improved

**Comment by NerminSalem** [comment]
hi, i have been trying to train with a small dataset with batch_size= 4 and it always perfrom 2 out of the 3 epochs in a really fast mode then stay still do nothing, it don't enter the 3rd epoch? what do u think may be the cause of that?
thanks for your consistent help
![screenshot from 2019-02-08 14-06-04](https://user-images.githubusercontent.com/26221951/52477479-b3a6d280-2baa-11e9-804e-f3ec3f4b474a.png)

**Comment by NerminSalem** [comment]
![screenshot from 2019-02-08 18-32-39](https://user-images.githubusercontent.com/26221951/52491506-f038f500-2bcf-11e9-84bc-7d47cf92a772.png)
i m trying to train with a small dataset composed of 100 images and 6 masks with batch_size=4 and it always complete 2/3 epochs and stay like this? any idea what to do?
thanks
